### PR TITLE
chore: cleaning

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationEndpointMock.cs [hunk 1]
```diff
@@ -59,26 +57,25 @@ private AuthenticationData CreateAuthenticationData(AuthenticationToken token =
...
+		const string signature = "QWqnPP8W6ymexz74P6quP-oG-wxr7vMGqrEL8y_tV6M"; // Dummy stuff.
 
 		var now = DateTimeOffset.Now;
 
-		token = (token ?? AuthenticationToken.Default) with
-		{
-			Expiration = now + (timeToLive ?? TimeSpan.FromMinutes(10)),
-			IssuedAt = now
-		};
+		token = new AuthenticationToken(
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationEndpointMock.cs:74]: That's not equivalent.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]
Do we need to do this refactoring? Is it the right moment?

File: src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationEndpointMock.cs (hunk 1 above)
Comment (Soap-141) [reply src/app/ApplicationTemplate.Client/ApiEndpoints/Authentication/AuthenticationEndpointMock.cs:74]: Forgot null check.

**Comment by Soap-141** [comment]
> Do we need to do this refactoring? Is it the right moment?

Fixed some annoying warning and it was fast.

**Pull request closed without merging**